tests/sentry/web/frontend/test_auth_organization_login.py

```diff
@@ -993,23 +992,13 @@ def test_flow_verify_and_link_without_password_sends_email(self, email, mock_red
         assert resp.status_code == 200
         assert resp.context["existing_user"] == self.user
 
-        ((name, time, value), kwargs) = mock_redis.return_value.setex.call_args
-        redis_obj = json.loads(value)
-        assert redis_obj["user_id"] == self.user.id
-        assert redis_obj["email"] == self.user.email
-        assert redis_obj["member_id"] == om.id
-        assert redis_obj["organization_id"] == self.organization.id
-        assert redis_obj["identity_id"]["id"] == self.user.email
-        assert redis_obj["provider"] == "dummy"
-
```

Comment:
This Redis mock was taking the place of a mocked reference to the `idpmigration.send_one_time_account_confirm_link` method. Now that we're exercising the back half of the flow, I don't think we need either.

We could keep the Redis mock in place, for the sake of these ☝️ more direct assertions on what's being inserted. But I think the `sentry-idp-email-verification` part adequately exercises the most important parts and it's more natural not to have to mock `mock_redis.return_value.get`.